docs/2023/Sunbird ED/Bulk content creation/updates/2023-07-28.md

```diff
@@ -4,13 +4,20 @@ author: Deepali Lodhi
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
+- [x] Added Progress bar in the upload feature
+- [x] Added common metadata field 
+- [x] Added multiple metadata field
+- [x] Add Dropzone library in bulkUploadContent project
+- [x] Optimize code
 
 ## Screenshots / Videos 
+- 
 
 ## Contributions
 
-## Learnings
\ No newline at end of file
+## Learnings
+- Learned Dropzone js library
+- Optimized code
+- Get to know about files-ui
+
+- Presentation Link (https://docs.google.com/presentation/d/1_VkKwYR0tvPDuIEKJ3xZJu1olZASjBrrJxIMnjGbh88/edit?usp=sharing)
```

Comment:
Add links in this format [PPT](linkToPPT) so that it is considered link and not plain text